flexbox/css-flexbox_profile/css/styles.css

```diff
@@ -14,15 +14,26 @@ body {
   background: #f3f5f9;
 }
 
+.profile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
This center's the all the content and not just the image.

Reply:
Yes I didn't know it was just the image that had to be centered.